cleanlab/models/keras.py

```diff
@@ -121,10 +124,12 @@ def predict_proba(self, X, *, apply_softmax=True, **kwargs):
         return pred_probs
 
     def predict(self, X, **kwargs):
+        """Predict class labels using the wrapped Keras classifier."""
```

Comment:
For all `predict()` and `predict_proba()` methods, should clarify that the new data `X` must be in same format as the original `X` provided to `fit()`?  

I assume user cannot train with TF dataset and then switch to numpy array for prediction

Reply:
Technically this is doable but I don't think we want users doing it, will add a statement to clarify

Reply:
Yep I think it's ok not to add too much. But isn't there currently no documentation for `X` in the predict() methods?
I was just suggesting something minimal like:

> X : data in the same format as the original \`X\` provided to \`\`fit()\`\`